packages/apollo-dx/rollup.config.js

```diff
@@ -1,20 +1,8 @@
 import resolve from 'rollup-plugin-node-resolve';
-import babel from 'rollup-plugin-babel';
```

Comment:
The original idea was to use the build/js files to avoid the babel plugin, but it does not work as it does not go into sub modules. So I guess we still need the rollup-plugin-babel. In all casses rollup build does not work and is excluded from the default build for now ..